src/translations/sqlb_nl.ts

```diff
@@ -0,0 +1,7344 @@
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../CondFormat.h" line="25"/>
+        <source>Left</source>
+        <translation>Links</translation>
+    </message>
+    <message>
+        <location filename="../CondFormat.h" line="25"/>
+        <source>Right</source>
+        <translation>Rechts</translation>
+    </message>
+    <message>
+        <location filename="../CondFormat.h" line="25"/>
+        <source>Center</source>
+        <translation>Gecentreerd</translation>
+    </message>
+    <message>
+        <location filename="../CondFormat.h" line="25"/>
+        <source>Justify</source>
+        <translation>Uitgevuld</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.cpp" line="82"/>
+        <source>All files (*)</source>
+        <translation>Alle bestanden (*)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="13"/>
+        <source>SQLite Database Files (*.db *.sqlite *.sqlite3 *.db3)</source>
+        <translation>SQLite-databasebestanden (*.db *.sqlite *.sqlite3 *.db3)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="16"/>
+        <source>DB Browser for SQLite Project Files (*.sqbpro)</source>
+        <translation>DB-browser voor SQLite-projectbestanden (*.sqbpro)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="20"/>
+        <source>SQL Files (*.sql)</source>
+        <translation>SQL-bestanden (*.sql)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="24"/>
+        <source>All Files (*)</source>
+        <translation>Alle bestanden (*)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="27"/>
+        <source>Text Files (*.txt)</source>
+        <translation>Tekstbestanden (*.txt)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="31"/>
+        <source>Comma-Separated Values Files (*.csv)</source>
+        <translation>Kommagescheiden bestanden (*.csv)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="33"/>
+        <source>Tab-Separated Values Files (*.tsv)</source>
+        <translation>Tabgescheiden bestanden (*.tsv)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="34"/>
+        <source>Delimiter-Separated Values Files (*.dsv)</source>
+        <translation>Scheidingstekengescheiden bestanden (*.dsv)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="35"/>
+        <source>Concordance DAT files (*.dat)</source>
+        <translation>Concordance-DAT-bestanden (*.dat)</translation>
+    </message>
+    <message>
+        <location filename="../FileDialog.h" line="38"/>
+        <source>JSON Files (*
```

Comment:
I think it's a proper noun that doesn't need translation, so I think you've left the translation blank.
I also did like this in the past, but there was an comments that it is safer to copy the original text for proper nouns
that do not require translation. Please refer to [here](https://github.com/sqlitebrowser/sqlitebrowser/pull/2329#issuecomment-658202890) for details.